Test/api-functional/GraphQl/AdyenTest.php

```diff
@@ -236,4 +334,20 @@ private function getPlaceOrderQuery(
 }
 QUERY;
     }
+
+    /**
+     * Create a header with customer token
+     *
+     * @param string $username
+     * @param string $password
+     * @return array
+     * @throws AuthenticationException
+     * @see \Magento\GraphQl\Quote\Customer\GetCustomerCartTest::getHeaderMap
+     */
+    private function getHeaderMap(string $username = 'customer@example.com', string $password = 'password'): array
+    {
+        $customerToken = $this->customerTokenService->createCustomerAccessToken($username, $password);
+        $headerMap = ['Authorization' => 'Bearer ' . $customerToken];
+        return $headerMap;
```

Comment:
Would be nicer to immediately return this like:
```return ['Authorization' => 'Bearer ' . $customerToken];```

Reply:
True, thanks for pointing it out!